plugins.txt

```diff
@@ -84,7 +84,7 @@ timestamper:1.13
 token-macro:266.v44a80cf277fd
 toolenv:1.2
 variant:1.4
-warnings-ng:9.4.0
+warnings-ng:9.5.0
```

Comment:
https://github.com/jenkinsci/warnings-ng-plugin/releases/tag/v9.5.0 => Deprecation of some. Let's see